vms/platformvm/txs/mempool/mempool.go

```diff
@@ -45,13 +47,15 @@ var (
 
 type Mempool interface {
 	Add(tx *txs.Tx) error
-	Has(txID ids.ID) bool
-	Get(txID ids.ID) *txs.Tx
-	Remove(txs []*txs.Tx)
+	Get(txID ids.ID) (*txs.Tx, bool)
+	Remove(txs ...*txs.Tx)
 
 	// Peek returns the oldest tx in the mempool.
 	Peek() (tx *txs.Tx, exists bool)
 
+	// Iterate iterates over the txs until f returns false
```

Comment:
Would it make sense to note that the txs are iterated FIFO?

Reply:
We can but the callers of `gossip.Set#Iterate` is agnostic to the iteration ordering... so we can add this comment or not but I don't see a reason to guarantee it if we don't have to

Reply:
ok